lightning-background-processor/src/lib.rs

```diff
@@ -50,7 +50,10 @@ const FRESHNESS_TIMER: u64 = 60;
 #[cfg(test)]
 const FRESHNESS_TIMER: u64 = 1;
 
+#[cfg(not(debug_assertions))]
 const PING_TIMER: u64 = 5;
+#[cfg(debug_assertions)]
+const PING_TIMER: u64 = 30;
```

Comment:
Can you add a comment here noting that 30 seconds on some devices results in signature operations taking long enough that we disconnect?